search-commons/src/test/resources/resource_datasource.json

```diff
@@ -162,7 +161,7 @@
         }
       }
     },
-    "identifier": "018ba3cfcb9c-94f77a1e-ac36-430a-84b0-0619ecbbaf39",
+    "identifier": "018ba3cfcb9c-94f77a1e-ac36-430a-84b0-0619e3bbaf39",
```

Comment:
This error has been here a long time... we had two equal indetifiers in this index, I thought it was filtered by status, it was not...